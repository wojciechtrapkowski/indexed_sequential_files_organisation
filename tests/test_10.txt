insert 3 300
insert 5 500
insert 4 400
insert 6 600
insert 9 900
insert 8 800
insert 7 700
reorganise
print
search 3
search 4
search 5
search 6
search 7
search 8
search 9
