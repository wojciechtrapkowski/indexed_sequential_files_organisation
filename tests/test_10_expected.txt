






================================================
Index area
================================================
Page 0 number of entries: 2
	Entry 0
		start_key: 3
		page_index: 0
	Entry 1
		start_key: 7
		page_index: 1
================================================
Main area
================================================
Guardian overflow page index: null

Page 0 number of entries: 4
	Entry 0
		key: 3
		value: 300
		overflow_entry_index: null
	Entry 1
		key: 4
		value: 400
		overflow_entry_index: null
	Entry 2
		key: 5
		value: 500
		overflow_entry_index: null
	Entry 3
		key: 6
		value: 600
		overflow_entry_index: null
Page 1 number of entries: 3
	Entry 0
		key: 7
		value: 700
		overflow_entry_index: null
	Entry 1
		key: 8
		value: 800
		overflow_entry_index: null
	Entry 2
		key: 9
		value: 900
		overflow_entry_index: null
================================================
Overflow area
================================================
Page 0 number of entries: 0
300
400
500
600
700
800
900
